en/topics.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Topics
+permalink: /en/topics/
+layout: page
+---
+{% include linkers/topic-pages.md %}
+
+<div>{% comment %}<!-- enclosing in a div forces this to be interpreted
+as HTML rather than Markdown so indentation over 4 characters doesn't
+produce code blocks -->{% endcomment %}
+
+{% comment %}<!-- Build an "ENDTOPIC"-separated string with
+Markdown-style links for each topic or topic alias.  We use
+Markdown-style links, e.g. [Name](URL), instead of HTML-style
+links, e.g. <a href=URL>Name</a>, so that it's easy to sort by name
+rather than URL. -->{% endcomment %}
+{% capture raw_topics_list %}
+{%- for topic in site.topics -%}
+  [{{topic.title}}]({{topic.url}})ENDTOPIC
+  {%- for alias in topic.aliases -%}
+    [{{alias}}]({{topic.url}})ENDTOPIC
+  {%- endfor -%}
+{%- endfor -%}
+{% endcapture %}
+
+{% assign topics_list = raw_topics_list | split: 'ENDTOPIC' | sort_natural %}
+{% for entry in topics_list %}
```

Comment:
It'd be great if the list could dynamically add columns when displayed on wider monitors. Just having a single column means there's a lot of white space and unnecessary scrolling.

Reply:
I did that for the developer glossary on Bitcoin.org ([here](https://btcinformation.org/en/developer-glossary) with my original layout) but I ended up thinking that it wasn't the best tradeoff.  Fill lists need to restrict the number of characters each term can have, they're more likely to fail on some browser, and they force the user to move their eyes excessively to read each term.  Simple linear lists do require more scrolling on widescreens, as you indicate, but that doesn't seem so bad to me---in places where people have widescreens, they probably also have the ability to scroll quickly (e.g. scroll wheels, scroll sliding, and page-down keys).

Reply:
Sounds good. One column seems reasonable.